### DISCUSSION TITLE
[Bug]: OAuth2Client Token

### DISCUSSION THREAD

**Discussion by marzhochi** [ivanwilliammd/satusehat-integration#133] [Q&A (Tanya Jawab)] [answered]
### What happened?

![Screenshot 2024-06-29 092229](https://github.com/ivanwilliammd/satusehat-integration/assets/74941854/571b9318-cbd5-46fa-b64c-cce44ee92e7b)

mau nanya dong. di saya beberapa kali ketika request OAuth2Client sering terjadi error seperti ini,  ini error dari SATUSEHATnya atau dari codingan pada proses get token kah, minta pencerahan dong.

### How to reproduce the bug

maaf masih pemula jadi blm bisa explore library nya. jd belum bisa memberi masukkan sumber kesalahannya.

### Package Version

2.7.0

### PHP Version

8.1.19

**Comment by ivanwilliammd** [comment, marked as answer]
Ini kendalan dari server staging SATUSEHAT (error 500), dan biasanya karena internet provider / server yang digunakan biasanya tidak static IP dari Indonesia. Silakan dicek dari infrastrukturnya ya
